src/main/scala/io/sdkman/changelogs/BTraceMigrations.scala

```diff
@@ -5,44 +5,18 @@ import com.mongodb.client.MongoDatabase
 
 @ChangeLog(order = "027")
 class BTraceMigrations {
-
-  @ChangeSet(order = "001", id = "001-add_btrace_2_0_1", author = "bachorikj")
+  @ChangeSet(
+    order = "001",
+    id = "001_add_btrace_candidate",
+    author = "bachorikj"
+  )
   def migration001(implicit db: MongoDatabase) = {
-    List(
-      Version(
-        "btrace",
-        "2.0.1",
-        "https://github.com/btraceio/btrace/releases/download/v2.0.1/btrace-bin.tar.gz",
-        Universal
-      ),
-      Version(
-        "btrace",
-        "2.0.1",
-        "https://github.com/btraceio/btrace/releases/download/v2.0.1/btrace-bin.zip",
-        Windows
-      )
-    ).validate()
-      .insert()
-
-    setCandidateDefault("btrace", "2.0.1")
-  }
-
-  @ChangeSet(order = "002", id = "002-add_btrace_1_3_11_3", author = "bachorik")
-  def migration002(implicit db: MongoDatabase) = {
-    List(
-      Version(
-        "btrace",
-        "1.3.11.3",
-        "https://github.com/btraceio/btrace/releases/download/v1.3.11.3/btrace-bin-1.3.11.3.tgz",
-        Universal
-      ),
-      Version(
-        "btrace",
-        "1.3.11.3",
-        "https://github.com/btraceio/btrace/releases/download/v1.3.11.3/btrace-bin-1.3.11.3.zip",
-        Windows
-      )
-    ).validate()
-      .insert()
+    Candidate(
+      candidate = "btrace",
+      name = "BTrace",
+      description =
+        "BTrace - a safe, dynamic tracing tool for the Java platform",
```

Comment:
Sorry for being a pain, but would you mind giving a longer description please? This appears on the website under the [SDKs page](https://sdkman.io/sdks) as well as the `sdk list` command.

Reply:
Ok. I put some more info there.

Reply:
Hi @marc0der  - does this look ok now?

Reply:
Yes, that looks a lot better. I'll merge this over the weekend and get back to you about the rest.